pypit/arproc.py

```diff
@@ -333,7 +363,10 @@ def bg_subtraction_slit(slf, det, slit, sciframe, varframe, crpix, tracemask=Non
         #                        **settings.argflag['reduce']['skysub']['bspline'])
         ivar = arutils.calc_ivar(varframe)
         mask, bspl = arutils.robust_polyfit(tilts[gdp][srt], scifrcp[gdp][srt], 3,
-                                            function='bspline', weights=np.sqrt(ivar)[gdp][srt], sigma=5., maxone=False, **settings.argflag['reduce']['skysub']['bspline'])
+                                            function='bspline',
+                                            weights=np.sqrt(ivar)[gdp][srt],
+                                            sigma=5., maxone=False,
+                                            **settings.argflag['reduce']['skysub']['bspline'])
```

Comment:
Not sure about this.  I think I'd prefer that you just pass it as a dict.  Something like `bspline_par = settings.argflag['reduce']['skysub']['bspline']` .

Reply:
Agreed.   This was a bit pernicious but now addressed.